scripts/init.sql

```diff
@@ -1,158 +1,147 @@
-        updated_at timestamp,
-        CONSTRAINT orders_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS users(
-        id integer NOT NULL,
-        first_name varchar(255) NOT NULL,
-        last_name varchar(255) NOT NULL,
-        email varchar(255) NOT NULL,
-        "password" varchar(100) NOT NULL,
-        "role" integer NOT NULL,
-        login_mode integer NOT NULL,
-        bio varchar(500),
-        verified boolean NOT NULL DEFAULT false,
-        image_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        deleted_at timestamp,
-        CONSTRAINT users_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS verification_tokens(
-        id integer NOT NULL,
-        token varchar(255) NOT NULL,
-        "type" integer NOT NULL,
-        expiration timestamp NOT NULL,
-        user_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        CONSTRAINT verification_tokens_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS verifications(
-        id integer NOT NULL,
-        email_verified boolean NOT NULL,
-        user_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT verifications_pkey PRIMARY KEY(id)
-    );
-
-CREATE TABLE
-    IF NOT EXISTS wishcards(
-        id integer NOT NULL,
-        address_line_1 varchar(255) NOT NULL,
-        address_line_2 varchar(255),
-        city varchar(100) NOT NULL,
-        state varchar(100) NOT NULL,
-        country varchar(100) NOT NULL,
-        zip_code varchar(50) NOT NULL,
-        status integer NOT NULL DEFAULT 0,
-        created_by integer NOT NULL,
-        agency_id integer NOT NULL,
-        child_id integer NOT NULL,
-        item_id integer NOT NULL,
-        image_id integer NOT NULL,
-        order_id integer NOT NULL,
-        created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-        updated_at timestamp,
-        CONSTRAINT wishcards_pkey PRIMARY KEY(id)
-    );
-
--- constraints
+SET check_function_bodies = false ;
+
+CREATE TABLE agencies(
+    id uuid NOT NULL DEFAULT gen_random_uuid(),
+    "name" varchar(255) NOT NULL,
+    website varchar(255) NOT NULL,
+    phone varchar(100) NOT NULL,
+    bio varchar(500),
+    address_line_1 varchar(255) NOT NULL,
+    address_line_2 varchar(255),
+    city varchar(100) NOT NULL,
+    state varchar(100) NOT NULL,
+    country varchar(100) NOT NULL,
+    zip_code varchar(50) NOT NULL,
+    verified boolean NOT NULL DEFAULT false,
+    employer_identification_number varchar(255) NOT NULL,
+    account_manager_id uuid NOT NULL,
+    image_id uuid NOT NULL,
+    created_at timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at timestamp,
+    CONSTRAINT agencies_pkey PRIMARY KEY(id),
+    UNIQUE(employer_identification_number)
```

Comment:
```suggestion
    UNIQUE(id),
    UNIQUE(employer_identification_number)
```

@Enubia UUID collisions are next to impossible, but how do you feel about adding a unique constraint to each of them so we can prevent a duplicate entry if it somehow ever were to happen?

Reply:
@stacysealky We should look into obtaining the EIN for all existing agencies for compliance reasons, honestly I think the best option is to remove the NOT NULL constraint until we can backfill the existing ones. I don't believe the `UNIQUE` constraint applies to `null` fields, so we should be fine there as long as that is the case.

Reply:
@nick-w-nick 
re: "We should look into obtaining the EIN for all existing agencies for compliance reasons"
-> I can look into it, but it will be kinda time consuming.

@Enubia we should remove the NOT NULL constraint for now